packages/expo-cli/src/commands/build-native/index.ts

```diff
@@ -88,7 +128,10 @@ export default function (program: Command) {
     .asyncActionProjectDir(buildAction, { checkConfig: true });
 
   program
-    .command('build:status')
+    .command('build-status [project-dir]')
+    .option('-p --platform <platform>')
+    .option('-s --status <status>')
+    .option('-b --build-id <build-id>')
```

Comment:
Please add descriptions for these options (also, list possible values for enum options), sth like: https://github.com/expo/expo-cli/blob/master/packages/expo-cli/src/commands/upload.ts#L27